recipes/subprocess32-ext/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "subprocess32-ext" %}
+{% set version = "3.2.8.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: f2cafa80ef9706bb8400339b9120820e03b07af5e9a88736ba3daca30a47371b
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py < 35 or (win and vc <= 14)]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - python3_redirect
+    - subprocess32
+
+about:
+  home: "https://github.com/google/python-subprocess32"
+  license: "PSF"
+  license_family: "OTHER"
+  license_file: "LICENSE"
+  summary: "A backport of the subprocess module from Python 3.2/3.3 for use on 2.x."
+  doc_url: "https://pypi.org/project/subprocess32-ext/"
+  dev_url: "https://github.com/google/python-subprocess32"
```

Comment:
Two separate entries on PyPl both link to the same github page
@jakirkham I'm unsure of the differences in the two; subprocess32-ext (rather than subprocess32) is listed as a dependency for another package I'm trying to add